src/database/migrations/20190703214332-create-article.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
```

Comment:
I think this file was already pushed before it was added to .gitignore. You may want to delete this file since you did not work on it. You can use `git rm src/database/migrations/20190703214332-create-article.js` to remove this and stop the file from being tracked.